include/CPushConsumer.h

```diff
@@ -36,6 +36,7 @@ ROCKETMQCLIENT_API CPushConsumer* CreatePushConsumer(const char* groupId);
 ROCKETMQCLIENT_API int DestroyPushConsumer(CPushConsumer* consumer);
 ROCKETMQCLIENT_API int StartPushConsumer(CPushConsumer* consumer);
 ROCKETMQCLIENT_API int ShutdownPushConsumer(CPushConsumer* consumer);
+ROCKETMQCLIENT_API const char* ShowPushConsumerVersion(CPushConsumer* consumer);
```

Comment:
It would be better if we could get build info and core version from the property or using some other automatic style like maven build plugin, print it on startup and provide the runtime command for RocketMQ CLI framework.